201-oms-extension-windows-vm/prereqs/azureprereqdeploy.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "omsWorkspacename": {
+            "type": "string",
+            "metadata": {
+                "description": "OMS Workspace name"
+            }
+        }
+    },
+    "variables": {
+
+    },
+    "resources": [{
+        "name": "[parameters('omsWorkspacename')]",
+        "type": "Microsoft.OperationalInsights/workspaces",
+        "apiVersion": "2015-11-01-preview",
+        "location": "East US",
+        "properties": {
+            "sku": {
+                "name": "Free"
+            }
+        }
+    }],
+    "outputs": {
+        "workSpaceName": {

```

Comment:
As of now, we name outputs in azureprereqdeploy.json same as the corresponding parameter name in azuredeploy.parameters.json for easy identification (Even though in this particular template it is not followed by mistake- **workspaceName** v/s **workSpaceName**
Should we continue with this approach or go with a consistent name?